app/views/content/a-day-in-the-life-of-a-teacher.md

```diff
@@ -1,7 +1,8 @@
 ---
-  title: "A day in the life of a teacher"
+  title: "What it's like to be a teacher"
```

Comment:
We will need to add this page to the navigation and into one of the life as a teacher sections. Perhaps either teaching as a career?